packages/libx265/build.sh

```diff
@@ -2,17 +2,35 @@ TERMUX_PKG_HOMEPAGE=http://x265.org/
 TERMUX_PKG_DESCRIPTION="H.265/HEVC video stream encoder library"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=3.3
+TERMUX_PKG_VERSION=3.5
 TERMUX_PKG_SRCURL=https://bitbucket.org/multicoreware/x265_git/downloads/x265_$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=c6d744a87eda55560da715f56f878640554ddc06e2d0fcbd822fa330affc22cc
+TERMUX_PKG_SHA256=e70a3335cacacbba0b3a20ec6fecd6783932288ebc8163ad74bcc9606477cae8
 TERMUX_PKG_DEPENDS="libc++"
 TERMUX_PKG_BREAKS="libx265-dev"
 TERMUX_PKG_REPLACES="libx265-dev"
 
 termux_step_pre_configure() {
-	if [ $TERMUX_ARCH = "i686" ]; then
+	local ARM_ARGS
+
+	# Not sure if this is necessary for on-device build
+	ARM_ARGS="-fno-integrated-as"
```

Comment:
Yes, this is required to invoke `GNU AS` for assembly code. On-device, one needs to install `binutils`.

This is a long shot, but whenever the next android-ndk release would drop support for `as`, I guess, only on-device builds will be possible. This will continue until the projects upstream update their `asm` code to be compatible with llvm. https://android.googlesource.com/platform/ndk/+/master/docs/ClangMigration.md

Reply:
We'll hopefully upgrade to ndk r23 (which is a LTS release) soon, I don't think we will be in a hurry to upgrade to r24 when it is released